app/assets/stylesheets/utilities/_overwrites.scss

```diff
@@ -78,3 +78,19 @@
     font-size: inherit;
   }
 }
+
+.govuk-link {
```

Comment:
We shouldn't need to nest these - a modifier class should not depend on the non-modified class to be present.